Sources/Tracking/CustomerIO.swift

```diff
@@ -20,7 +24,7 @@ public protocol CustomerIOInstance: AutoMockable {
         // sourcery:TypeCast="AnyEncodable(body)"
         body: RequestBody
     )
-
+    func registeredDeviceToken() -> String?
```

Comment:
🚧 (blocker comment)

```suggestion
    var registeredDeviceToken: String? { get }
```

I prefer that we use a property accessor like we do for other property getters such as `CustomerIO.shared.config`. The API is more Swift-y.

Reply:
Thanks for the suggestion. But I do not understand how this is a blocker ? This is a suggestion and shouldn't be called out as a blocker IMO.